### PR TITLE
first batch of bugfixing on templates

### PR DESCRIPTION
**Pull request by voisardf**
This PRs will address the issues of https://github.com/camptocamp/pyramid_oereb/issues/398
more to come next week

### CODE REVIEW

File: print/print-apps/oereb/glossar.jrxml [hunk 1]
```diff
@@ -132,7 +132,7 @@
 				<textElement>
 					<font fontName="Cadastra" size="6.5"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$P{CreationDate}]]></textFieldExpression>
+				<textFieldExpression><![CDATA[$P{CreationDate}.toString().substring(8,10)+'.'+$P{CreationDate}.toString().substring(5,7)+'.'+$P{CreationDate}.toString().substring(0,4)+' '+$P{CreationDate}.toString().substring(11,13)+':'+$P{CreationDate}.toString().substring(14,16)+':'+$P{CreationDate}.toString().substring(17,19)]]></textFieldExpression>
```
Comment (sbrunner) [inline print/print-apps/oereb/glossar.jrxml]: What's this incomprehensible thing...

Shouldn't we use something like:
```
new java.text.SimpleDateFormat("HH mm ss dd MM yyyy").format(new java.text.SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss").parse($P{CreationDate}))
```

To be more understandable ...

Comment (sbrunner) [reply print/print-apps/oereb/glossar.jrxml]: Or better:
```
java.time.format.DateTimeFormatter.ofPattern("HH mm ss dd MM yyyy").format(java.time.LocalDateTime.parse($P{CreationDate}))
```

Comment (voisardf) [reply print/print-apps/oereb/glossar.jrxml]: Thanks, I'm absolutely not familiar with the java functionnality in Jasper, so your suggestion is perfect for me. I'll put it in and push an update 👍

**Comment by voisardf** [comment]
@sbrunner : changed the date as suggested
@vvmruder @kdeininger : that should also address Issue https://github.com/camptocamp/pyramid_oereb/issues/411

**Comment by voisardf** [comment]
just added the FOS number to the french ressources to answer to complet the german fix in PR https://github.com/camptocamp/pyramid_oereb/pull/421

**Pull request merged**